exasol/nb_connector/slc/script_language_container.py

```diff
@@ -246,10 +246,9 @@ def docker_image_tags(self) -> list[str]:
 
     def clean_docker_images(self):
```

Comment:
Make a static function , I think, we can use as output dir the current working directory

Reply:
Hm, we  need the `checkout_dir`  and ` workspace` variables from self. I don't think a static function would be wise....